Sources/SwiftGenKit/Stencil/AssetsCatalogContext.swift

```diff
@@ -42,9 +42,10 @@ extension AssetsCatalog.Parser {
   private func structure(entries: [AssetsCatalog.Entry]) -> [[String: Any]] {
     return entries.map { entry in
       switch entry {
-      case .group(let name, let items):
+      case .group(let name, let isNamespaced, let items):
         return [
           "type": "group",
+          "isNamespaced": "\(isNamespaced)",
```

Comment:
Stencil can handle booleans, no need to stringify the value here.

Reply:
It's harder than it seems.
It does work with Stencil but it breaks Yaml deserialization tests. Because tests are using Yaml `.basic` (so everything is string). So true would never equal "true".
So basically I had to use strings to ensure tests are passing :(

Reply:
Hmmm, that's not the first time we've encountered issues with YAML deserialization. Lemme check

Reply:
We could switch to non-basic parsing, but that would break one of our `strings` unit tests where a string key has as value "no":
https://github.com/SwiftGen/SwiftGen/blob/master/Tests/Fixtures/StencilContexts/Strings/multiple.yaml#L37

Ideally Yams'd have a mode to write quotes around strings, but not annotate all the other types (which is what `canonical` does I think? Can't remember, it's been a while).